tft.py

```diff
@@ -211,19 +213,27 @@ def queue() -> None:  # pylint: disable=too-many-branches
             break
 
         if LCU_INTEGRATION.in_queue():
-            time.sleep(5)
-            if LCU_INTEGRATION.found_queue():
+            start_queue_repeating = False
+            if LCU_INTEGRATION.found_queue() and not LCU_INTEGRATION.queue_accepted():
                 LCU_INTEGRATION.accept_queue()
-                time.sleep(10)
-                continue
+                time.sleep(3)
+            else:
+                time.sleep(3)
             continue
 
         if not PLAY_NEXT_GAME:
             continue
 
         if LCU_INTEGRATION.in_lobby():
+            # Fix for if the user is considered not ready.
+            if start_queue_repeating:
+                LCU_INTEGRATION.delete_lobby()
+                start_queue_repeating = False
+                time.sleep(1)
+                continue
+
             LCU_INTEGRATION.start_queue()

```

Comment:
Is there any chance we need a slight delay here to wait for the `start_queue` to have started before it loops around for the next `in_queue` check, or is it blocking and that shouldn't happen?

Reply:
It should be blocking since it's a synchronous request, but probably a good idea to just wait for a second anyway.